ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/checkoutform.html

```diff
@@ -121,16 +121,31 @@
 					<p class ='error-message'></p>
 					</div>
 
-                    <div>
-					<!-- <input type="text" 
+                    <!--<div>
```

Comment:
@nitheshkumar-hcl : can we remove unwanted code from here

Reply:
Committed the changes as per the review comments